Sources/HexGrid/DataStructures/Point.swift

```diff
@@ -1,14 +1,13 @@
-public struct Point: Codable {
+/// Basic structure containing an `x` and `y` value.
+public struct Point: Codable, Equatable {
     public var x, y: Double
 
+    /// Initializing the `Point`.
+    /// - Parameters:
+    ///   - x: horizontal coordinate value
+    ///   - y: vertical coordinate value
     public init (x: Double, y: Double) {
         self.x = x
         self.y = y
     }
 }
-
-extension Point: Equatable {
```

Comment:
I guess there was a third change here in that I removed this custom `Equatable`. I think it's better to let Swift generate it, especially for this kind of simple structure.